app/services/youtube_notification_service.rb

```diff
@@ -31,6 +32,12 @@ def channels_for_project project
     project.slack_channel_codes
   end
 
+  def channels_for_event event_id
+    return [] if event_id.nil?
+    event = Event.find(event_id)
+    event.slack_channel_codes
+  end
+
```

Comment:
hmmm this bit of code seems like it could live in the Event model?

Reply:
this method `slack_channel_codes` is in the Event model...
it calls `slack_channels.pluck(:code)`

Reply:
or do you mean the entire method `channel_for_event`?

Reply:
Yes, I mean the entire method. The method calls the Event model and has an `event id` parameter so it seems that it should be in the Event model or perhaps event instance....

Reply:
Also if the method is in the model then the duplication in `hangout_notification_service.rb` can be removed.

Reply:
yes, great advice! I was saying to @gr1d99 that maybe we should be extracting into a helper method, but I like the idea of add it to the Event model

we could do something similar with
```
def channels_for_project project
    return [] if project.nil? or project.slug.nil?
    project.slack_channel_codes
  end
```
maybe create a separate ticket to refactor it.